lib/oauth.js

```diff
@@ -36,7 +36,7 @@ function oauth() {
       response["message"] = "Previous Token is still valid.";
     }
 
-    response["token"] = application_token["access_token"];
+    response["token"] = application_token["token"];
```

Comment:
Is error handling added here?

Reply:
Yes, it is handled earlier in fetchNewAccessToken function. If the token is already present it will run fine, or else if it fails while fetching new token, it will return failure message.